custom_components/hildebrandglow/glow.py

```diff
@@ -1,31 +1,54 @@
 """Classes for interacting with the Glowmarkt API."""
+from __future__ import annotations
+
 from pprint import pprint
-from typing import Any, Dict, List
+from typing import TYPE_CHECKING, Any, Dict, List
 
+import paho.mqtt.client as mqtt
 import requests
 from homeassistant import exceptions
 
+from .mqttpayload import MQTTPayload
+
+if TYPE_CHECKING:
+    from .sensor import GlowConsumptionCurrent
+
 
 class Glow:
     """Bindings for the Hildebrand Glow Platform API."""
 
     BASE_URL = "https://api.glowmarkt.com/api/v0-1"
+    HILDEBRAND_MQTT_HOST = "glowmqtt.energyhive.com"
+    HILDEBRAND_MQTT_TOPIC = "SMART/HILD/{hardwareId}"
```

Comment:
Would it be possible to make the MQTT topic configurable as well?  Happy with HILD being the default, but for some reason I've got a legacy account and I need to use `SMART/DCAD/{hardwareId}`

Maybe name it `mqtt_topic_prefix`, that has a default value of `HILD`?

Reply:
Hey @bje 👋 Thanks for raising this! That's an interesting edge case. I wonder if there's some way to automatically discover the right topic.

In the meantime, it seems there aren't any issues from subscribing to *both* topics (and only receiving messages from one), so I've put that in db42193.

Reply:
btw- I've opened a post on the official forums seeking advice on properly discovering the MQTT topic https://forum.glowmarkt.com/index.php?p=/discussion/75/mqtt-topic-variations